src/System.Data.Common/src/System/Data/SQLTypes/SQLBinary.cs

```diff
@@ -11,7 +11,6 @@
 
 namespace System.Data.SqlTypes
 {
-    [Serializable]
```

Comment:
I kept the equivalents of all of the primitives and I'll add SqlGuid in the next iteration. The rest are less obviously useful and we have no data supporting serializing them.